InGreedIoApi/Data/Seed/ApiUserSeeder.cs

```diff
@@ -1,27 +1,30 @@
 ﻿using InGreedIoApi.POCO;
 using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
 
 namespace InGreedIoApi.Data.Seed
 {
     public class ApiUserSeeder : IUserSeeder
     {
         private readonly UserManager<ApiUserPOCO> _userManager;
         private readonly RoleManager<IdentityRole> _roleManager;
+        private readonly IConfiguration _configuration;
 
-        public ApiUserSeeder(UserManager<ApiUserPOCO> userManager, RoleManager<IdentityRole> roleManager)
+        public ApiUserSeeder(UserManager<ApiUserPOCO> userManager, RoleManager<IdentityRole> roleManager, IConfiguration configuration)
         {
             _userManager = userManager;
             _roleManager = roleManager;
+            _configuration = configuration;
         }
 
         public async Task SeedAsync()
         {
-            var users = new List<(string username, string email, string password, string role)>
+            var users = new List<(string username, string email, string role)>
             {
-                ("bartek@example.com", "bartek@example.com", "Password123!", "User"),
-                ("bartosz@example.com", "bartosz@example.com", "Password123!", "Admin"),
-                ("bmajosek@example.com", "bmajosek@example.com", "Password123!", "Moderator"),
-                ("bmajos@example.com", "bmajos@example.com", "Password123!", "Producer")
+                ("bartek@example.com", "bartek@example.com", "User"),
+                ("bartosz@example.com", "bartosz@example.com", "Admin"),
+                ("bmajosek@example.com", "bmajosek@example.com", "Moderator"),
+                ("bmajos@example.com", "bmajos@example.com", "Producer")
```

Comment:
Can you change this emails to tell the role? ex. admin@example.com if it is admin. It would be better for presentation purposes on Monday. Remember to change it in appsettings.json